data/crac/crac-api/src/main/java/com/farao_community/farao/data/crac_api/Range.java

```diff
@@ -0,0 +1,11 @@
+package com.farao_community.farao.data.crac_api;
+
+public interface Range {
```

Comment:
What do you think about putting also PstRange and RangeDefinition in crac-api ? (see previous comment)

Reply:
seems like a good idea, RangeDefinition is already in the API so PstRange has good reasons to be too